addons/medical_treatment/stringtable.xml

```diff
@@ -1407,7 +1407,7 @@
             <Chinese>生理食鹽水 (250毫升)</Chinese>
         </Key>
         <Key ID="STR_ACE_Medical_Treatment_QuikClot_Display">
-            <English>Basic Field Dressing (QuikClot)</English>
+            <English>Bandage (QuikClot)</English>
             <Russian>Первичный перевязочный пакет (QuikClot)</Russian>
             <Spanish>Vendaje básico (QuickClot)</Spanish>
             <French>Pansement hémostatique (Quikclot)</French>
```

Comment:
```xml
             <German>Bandage (QuikClot)</German>
```
Sadly can't do a suggestion thing, because German is too deep down for this one. :(